lib/SIL/SILPrinter.cpp

```diff
@@ -1212,6 +1212,12 @@ class SILPrinter : public SILInstructionVisitor<SILPrinter> {
       *this << SILType::getPrimitiveObjectType(metatype.getInstanceType());
       return;
     }
+    case SymbolicValue::Function: {
+      auto function = v.getFunctionValue().first;
```

Comment:
Nice and simple, I like it.  Please assert that the conformance in ".second" isn't present.

Reply:
Done in https://github.com/apple/swift/pull/17374/commits/99ce3142f6b79bf99084768697f3f07cfe1f380b.